tsan_arangodb_suppressions.txt

```diff
@@ -24,6 +30,19 @@ race:VersionSet::SetLastSequence
 
 # TODO - this should be removed once BTS-685 is fixed
 race:AllowImplicitCollectionsSwitcher
+race:graph::RefactoredTraverserCache::appendVertex
```

Comment:
What kind of race is this? Can we fix it instead of suppressing it?

Reply:
It's same as above, just don't write switcher in call stack